src/VolumeOfFluidEquationSystem.C

```diff
@@ -117,7 +117,8 @@ VolumeOfFluidEquationSystem::VolumeOfFluidEquationSystem(
     smooth_(smooth),
     smoothIter_(smoothIter),
     isInit_(true),
-    scsAdvection_(false)
+    scsAdvection_(false),
+    buoyancyStab_(realm_.solutionOptions_->buoyancyPressureStab_ ? 1.0 : 0.0)
```

Comment:
I don't think you ever use this... As such, we should remove it solely in favor of using solutionOptions member data. Let me know if I am wrong.

Reply:
I believe this is leftover from when I was feeding in the value into the individual kernels/algorithms. It should be removed without issue. I'll do so here in a moment and make sure it all still works

Reply:
This has now been removed in latest push.